### DISCUSSION TITLE
NodeJS CPU Profiling

### DISCUSSION THREAD

**Discussion by KrishnaJyothika** [grafana/pyroscope#2718] [Q&A]
Hi Team,

For NodeJS SDK we're seeing only memory and wall profiles in Pyroscope, we're are not getting CPU profiles.

<img width="901" alt="image" src="https://github.com/grafana/pyroscope/assets/121208944/d053b8c1-027a-419b-8da6-276ba1150ed0">

In logs we're seeing CPU profiles are collected

<img width="752" alt="image" src="https://github.com/grafana/pyroscope/assets/121208944/21b35aac-4c36-4410-9687-ca44c678da87">

Please let us know if this is the expected behavior with NodeJS SDK

**Comment by korniltsev** [comment]
this is expected until https://github.com/grafana/pyroscope-nodejs/pull/19 or https://github.com/grafana/pyroscope-nodejs/pull/39/files is merged

**Reply by KrishnaJyothika** [reply]
Thanks for the quick response @korniltsev 

Can you please let me know if there is any ETA for this?

**Reply by korniltsev** [reply]
we're not working on #19 and I dont know eta, and #39 is an external contribution so also dont know eta

**Reply by KrishnaJyothika** [reply]
Thanks for the info @korniltsev
